### ISSUE TITLE
ToQueryDefinition throws ArgumentNullException on empty query

### ISSUE THREAD

**Issue by joshlang** [Azure/azure-cosmos-dotnet-v3#719]
Using 3.2.0-preview

Calling `.ToQueryDefinition()` throws an ArgumentNullException with message `Value cannot be null. (Parameter 'sqlQuery')` if the query is empty.

Repo:
```
new CosmosClient("...connection string here...")
    .GetDatabase("asdf")
    .GetContainer("asdf")
    .GetItemLinqQueryable<object>()
    .ToQueryDefinition();
```

I would have expected this to produce text like `SELECT VALUE root FROM root`

But if I add a `.Where(...)` or anything before ToQueryDefinition, it works just fine.

**Comment by j82w** [comment]
Fixed in 3.3.0

**Comment by matthewhancock** [comment]
I am hitting this issue in 3.6.0.
